elements/d-d-d/package.json

```diff
@@ -47,6 +47,9 @@
     "@haxtheweb/hax-iconset": "^9.0.6",
     "@haxtheweb/simple-colors": "^9.0.6",
     "@haxtheweb/simple-icon": "^9.0.6",
+    "@haxtheweb/page-section": "^9.0.6",
+    "@haxtheweb/haxcms-elements": "^9.0.6",
+    "@haxtheweb/simple-modal": "^9.0.6",
```

Comment:
@klh6157 make these 3 dependencies show up under devDependencies. This is that weird package where we don't want ppl needing to install these just for the demo to work when it would invoke a ridiculous downstream series of dependencies (that will never get used) just so that we can demo DDD. I will make a new issue about splitting these out because of how absurd it is :)

Reply:
Yea, on Friday I will fix that. Which other elements should fall into this category? or do you think I should still put the others into dependency for now?

Reply:
oh ok, thats good to know